docs/examples.rst

```diff
@@ -309,6 +309,174 @@ limited for m2m fields. Also see below under optimization examples.
     as much as possible.
 
 
+Multi Table Inheritance
+-----------------------
+
+.. |br| raw:: html
+
+   <br />
+
+
+Multi table inheritance works with computed fields with some restrictions you have to be aware of.
+The following requires basic knowledge about multi table inheritance in Django and its similarities
+to o2o relations on accessor level (also see official Django docs).
```

Comment:
nit: maybe a link to the Django doc ?
https://docs.djangoproject.com/en/3.0/topics/db/models/#multi-table-inheritance